<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>872</width>
    <height>667</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Papagayo</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_6">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QScrollArea" name="scrollArea">
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAsNeeded</enum>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="WaveformView" name="waveformView">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>542</width>
           <height>194</height>
          </rect>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string>Current Voice</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Voice name:</string>
             </property>
             <property name="buddy">
              <cstring>voiceName</cstring>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="voiceName"/>
           </item>
           <item>
            <spacer name="horizontalSpacer_4">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Spoken text:</string>
           </property>
           <property name="buddy">
            <cstring>voiceText</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPlainTextEdit" name="voiceText">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Phonetic breakdown:</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QComboBox" name="languageChoice">
             <item>
              <property name="text">
               <string>English</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Spanish</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Italian</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="breakdownButton">
             <property name="text">
              <string>Breakdown</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QComboBox" name="exportChoice">
             <property name="editable">
              <bool>false</bool>
             </property>
             <item>
              <property name="text">
               <string>Anime Studio</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="exportButton">
             <property name="text">
              <string>Export...</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>FPS:</string>
          </property>
          <property name="buddy">
           <cstring>fpsEdit</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="fpsEdit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>1</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>Mouth 1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Mouth 2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Gary C Martin</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Preston Blair</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="MouthView" name="mouthView" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>200</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string>Voice List</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QListWidget" name="voiceList">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="editTriggers">
            <set>QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed</set>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QPushButton" name="newVoiceButton">
             <property name="text">
              <string>New</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="deleteVoiceButton">
             <property name="text">
              <string>Delete</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>872</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout_Papagayo"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="movable">
    <bool>true</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="separator"/>
   <addaction name="actionPlay"/>
   <addaction name="actionStop"/>
   <addaction name="separator"/>
   <addaction name="actionZoomIn"/>
   <addaction name="actionZoomOut"/>
   <addaction name="actionAutoZoom"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="../rsrc/papagayo.qrc">
     <normaloff>:/images/images/open.png</normaloff>:/images/images/open.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="../rsrc/papagayo.qrc">
     <normaloff>:/images/images/save.png</normaloff>:/images/images/save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save As...</string>
   </property>
  </action>
  <action name="actionPlay">
   <property name="icon">
    <iconset resource="../rsrc/papagayo.qrc">
     <normaloff>:/images/images/play.png</normaloff>:/images/images/play.png</iconset>
   </property>
   <property name="text">
    <string>Play</string>
   </property>
   <property name="toolTip">
    <string>Play</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="icon">
    <iconset resource="../rsrc/papagayo.qrc">
     <normaloff>:/images/images/stop.png</normaloff>:/images/images/stop.png</iconset>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
   <property name="toolTip">
    <string>Stop</string>
   </property>
  </action>
  <action name="actionZoomIn">
   <property name="icon">
    <iconset resource="../rsrc/papagayo.qrc">
     <normaloff>:/images/images/zoom_in.png</normaloff>:/images/images/zoom_in.png</iconset>
   </property>
   <property name="text">
    <string>Zoom In</string>
   </property>
   <property name="toolTip">
    <string>Zoom In</string>
   </property>
  </action>
  <action name="actionZoomOut">
   <property name="icon">
    <iconset resource="../rsrc/papagayo.qrc">
     <normaloff>:/images/images/zoom_out.png</normaloff>:/images/images/zoom_out.png</iconset>
   </property>
   <property name="text">
    <string>Zoom Out</string>
   </property>
   <property name="toolTip">
    <string>Zoom Out</string>
   </property>
  </action>
  <action name="actionAutoZoom">
   <property name="icon">
    <iconset resource="../rsrc/papagayo.qrc">
     <normaloff>:/images/images/zoom_1.png</normaloff>:/images/images/zoom_1.png</iconset>
   </property>
   <property name="text">
    <string>Auto Zoom</string>
   </property>
   <property name="toolTip">
    <string>Auto Zoom</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="text">
    <string>Cut</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="text">
    <string>Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionAbout_Papagayo">
   <property name="text">
    <string>About Papagayo</string>
   </property>
   <property name="toolTip">
    <string>About Papagayo</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>WaveformView</class>
   <extends>QWidget</extends>
   <header>waveformview.h</header>
   <container>1</container>
   <slots>
    <slot>onZoomIn()</slot>
    <slot>onZoomOut()</slot>
    <slot>onAutoZoom()</slot>
   </slots>
  </customwidget>
  <customwidget>
   <class>MouthView</class>
   <extends>QWidget</extends>
   <header>mouthview.h</header>
   <container>1</container>
   <slots>
    <slot>onMouthChanged(int)</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../rsrc/papagayo.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionOpen</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>onFileOpen()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>429</x>
     <y>334</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionPlay</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>onPlay()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>429</x>
     <y>334</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionStop</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>onStop()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>429</x>
     <y>334</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>newVoiceButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>onNewVoice()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>712</x>
     <y>617</y>
    </hint>
    <hint type="destinationlabel">
     <x>912</x>
     <y>581</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>deleteVoiceButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>onDeleteVoice()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>841</x>
     <y>617</y>
    </hint>
    <hint type="destinationlabel">
     <x>909</x>
     <y>680</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>voiceName</sender>
   <signal>textChanged(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>onVoiceNameChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>322</x>
     <y>324</y>
    </hint>
    <hint type="destinationlabel">
     <x>897</x>
     <y>440</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>voiceText</sender>
   <signal>textChanged()</signal>
   <receiver>MainWindow</receiver>
   <slot>onVoiceTextChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>541</x>
     <y>551</y>
    </hint>
    <hint type="destinationlabel">
     <x>892</x>
     <y>506</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>voiceList</sender>
   <signal>itemClicked(QListWidgetItem*)</signal>
   <receiver>MainWindow</receiver>
   <slot>onVoiceSelected(QListWidgetItem*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>704</x>
     <y>396</y>
    </hint>
    <hint type="destinationlabel">
     <x>900</x>
     <y>353</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>voiceList</sender>
   <signal>itemChanged(QListWidgetItem*)</signal>
   <receiver>MainWindow</receiver>
   <slot>onVoiceItemChanged(QListWidgetItem*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>780</x>
     <y>401</y>
    </hint>
    <hint type="destinationlabel">
     <x>916</x>
     <y>391</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>breakdownButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>onBreakdown()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>247</x>
     <y>617</y>
    </hint>
    <hint type="destinationlabel">
     <x>910</x>
     <y>279</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBox</sender>
   <signal>activated(int)</signal>
   <receiver>mouthView</receiver>
   <slot>onMouthChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>654</x>
     <y>121</y>
    </hint>
    <hint type="destinationlabel">
     <x>659</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSave</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>onFileSave()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>435</x>
     <y>333</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSave_As</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>onFileSaveAs()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>435</x>
     <y>333</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>exportButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>onExport()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>495</x>
     <y>600</y>
    </hint>
    <hint type="destinationlabel">
     <x>564</x>
     <y>593</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAbout_Papagayo</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>onHelpAboutPapagayo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>435</x>
     <y>333</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onFileOpen()</slot>
  <slot>onFileSave()</slot>
  <slot>onFileSaveAs()</slot>
  <slot>onPlay()</slot>
  <slot>onStop()</slot>
  <slot>onNewVoice()</slot>
  <slot>onDeleteVoice()</slot>
  <slot>onVoiceSelected(QListWidgetItem*)</slot>
  <slot>onVoiceNameChanged()</slot>
  <slot>onVoiceTextChanged()</slot>
  <slot>onVoiceItemChanged(QListWidgetItem*)</slot>
  <slot>onBreakdown()</slot>
  <slot>onExport()</slot>
  <slot>onHelpAboutPapagayo()</slot>
 </slots>
</ui>
